_posts/2022-09-21-hvr-and-hva.markdown

```diff
@@ -0,0 +1,110 @@
+
+- Consolidation (many-to-one)
+  This Topology involves multiple source locations consolidating data into a single target location.
+
+- Cascading (one-to-one-to-many)
+  This involves a source location pushing data to a target location that acts as a source location distributing data to multiple targets.
+
+- Bi-directional (active-active)
+  In this topology data is replicated in both directions and modified on both sides. It is referred to as an active/active scenario because the two sides are kept in sync. HVR employs the use of collision detection and loop-back detection to protect the integrity of the data
+
+- Multi-directional
+  Multi-directional replication involves more than two locations that are in an active-active replication setup.
+
+### Agents, Source Machines and Target Machines
+
+On each source/target machine (database, data warehouse or file) an agent can be installed and configured. The agent performs the capturing/integration of data that is consumed/processed by the HVR Hub Machine.
+The option to go agentless also exists, in which the HVR Hub Machine accesses a remote database/data warehouse via a DBMS protocol. This is generally not recommended due to the following key benefits provided by utilising the agent:
+
+- Reduction in network costs, distribution of CPU load and capturing of changes direct from DBMS logs
+- Compression of data before prior to sending to the hub, resulting in significant bandwidth savings (in ratios of up to 10x)
+- Secure and consistent connections that utilise encryption and authentication
+
+### User Interface and Rest API
+
+The two main methods in which a user can interact with the HVR Hub Server are Web UI and Remote CLI.
+The Web UI provides the means to visualise and control the replication process via a dashboard.
+
+The CLI can be accessed on the hub machine or from a remote machine provided there exists an HVR installation on the remote machine.
+In addition to the Web UI and CLI there exists a Rest API that can be used to build custom interactions in any programming language.
+
+## HVA
+
+### Overview
+
+High Volume Agent (HVA) is the marriage of HVR’s replication capabilities with Fivetran’s ease of use. HVR was recently acquired by Fivetran, and in line with Fivetran’s objectives, HVA hopes to house its replication technology within Fivetran’s core product. Presently, HVA is an enterprise ready Fivetran connector for Oracle and MySQL. The throughput of this replication is a staggering 10+ mB/s. The major use case for this tool is for enterprise companies that produce a large amount of data or have large amounts of historical data, that wish to transport its data and leverage one of the many cloud based storage solutions Fivetran integrates with.
+
+### Issues solved by HVA
+
+- Faster and higher volume replication
```

Comment:
compared to fivetran?

Reply:
Limitations of the HVA oracle connector?

NO TDE support
Interval data type  not supported

Reply:
I know HVR has no TDE, but I can't find any mention of this limitation in HVA setup or documentation. Could you link me ?

Reply:
I can't find any mention of TDE in HVA's docs or setup guide... is this online or from the training?

Reply:
was just in my notes, cant find docs on it